Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -110,6 +110,13 @@ public sealed class VsProjectAnalyzer : ProjectAnalyzer, IDisposable {
             IPythonInterpreterFactory factory,
             bool inProcess = false
         ) {
+            services.Python?.Logger?.LogEvent(PythonLogEvent.AnalysisInitializing, new AnalysisInitialize() {
```

Comment:
@huguesv  Can you add catch-all handlers around these? I'd hate to have something break because of logging

Reply:
done: https://github.com/Microsoft/PTVS/pull/3527